net-analyzer/linkchecker/linkchecker-9999.ebuild

```diff
@@ -21,6 +21,7 @@ IUSE="gnome sqlite"
 RDEPEND="
 	virtual/python-dnspython[${PYTHON_USEDEP}]
 	gnome? ( dev-python/pygtk:2[${PYTHON_USEDEP}] )
+	dev-python/pyxdg[${PYTHON_USEDEP}]
```

Comment:
please keep it sorted

Reply:
for readability, we also sort DEPENDs and RDEPENDs lexicographically, with unconditional dependencies preceding (USE-)conditional deps.

Reply:
Thank you for the explanation, now I understand!